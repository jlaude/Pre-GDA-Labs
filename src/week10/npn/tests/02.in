10
0
1.5
-1.5
- 0 1.5
+ + + 1.0 2.0 3.0 4.0
^ 2 5
* 2 5
/ 2 5
/ 2 0.1
* -5 -10
