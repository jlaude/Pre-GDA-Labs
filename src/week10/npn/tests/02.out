Case #1: 0.0
Case #2: 1.5
Case #3: -1.5
Case #4: -1.5
Case #5: 10.0
Case #6: 32.0
Case #7: 10.0
Case #8: 0.4
Case #9: 20.0
Case #10: 50.0
